pkgs/development/compilers/miranda/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, gcc }:
+
+stdenv.mkDerivation rec {
+  version = "2.066";
+  name = "${version}";
+
+  src = builtins.fetchTarball {
+    url = "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-2066-src.tgz";
+    sha256 = "0d16ibf3c3j93bx781bz3ga8q15acmb9080haixxjnsqpx6cakr8";
+  };
+
+  enableParallelBuilding = true;
+
+  configurePhase = ''
+    make cleanup
+    patchShebangs quotehostinfo
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    mkdir -p $out/share/man/man1
+    cp mira $out/bin
+    cp mira.1 $out/share/man/man1
+    cp -pPR miralib $out/lib
+  '';
+
+  buildInputs = [ gcc ];
```

Comment:
`gcc` is always there. As for your unfound libraries, try: `dontStrip = true;`.

Reply:
`gcc` may not be there on darwin, I'll make it dependent on the OS.

Reply:
Looks like `dontStrip = true;` didn't work, might be linked a different way?

Reply:
It's hard to tell, you'll need to dive into their build system a bit to figure this out.

Reply:
Maybe `gccStdenv` would then be a proper solution but not sure.